README.MD

```diff
@@ -0,0 +1,57 @@
+# Hermes-chat-v2-BE
+
+Hermes-chat-v2-BE represent server-side for a real-time messaging app, made with Express.js and Sequelize ORM
+Hermes-chat-v2-FE or the client side can be found on Github: https://github.com/BogdanMilivojevic/Hermes-chat-v2-FE
+
+Features:
+
+- Websocket enables real-time messaging
+- AWS's S3 enables image sharing between users
+- CI-CD pipeline which makes test passing a requirement for a successful merge
+- Authentification and Authorization
+- Tests are made with jest
+
+## Installation:
+
+### Step 1
+
+Clone the repository from GitHub: https://github.com/BogdanMilivojevic/Hermes-chat-v2-BE.git
+You will need node v18.9.0 and npm installed on your machine.
+
+### Step 2
+
+Open the terminal, navigate to the project folder and run the following comand:
+
+`$ docker-compose up --build`
+
+### Step 3
+
+Then, to create a database, in separate terminal run:
+
+`$ docker-compose exec app db:create`
+
+run migrations:
+
+`$ docker-compose exec app db:migrate`
+
+run seeders:
+
+`$ docker-compose exec app db:seed:all`
+
+### Step 4
+
+`$ docker-compose exec app npm install`
+
+### Step 5
+
+The web app should be available at localhost:4000
+
+## Deployment
+
+The app is deployed on Digital Space's Droplet
+
+Deploy is done automatically when merging into master.
+
```

Comment:
Add a section that explains how to run the current test suite, and how to add new tests.